src/coreclr/src/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.cs

```diff
@@ -13,8 +13,7 @@ internal static partial class GlobalizationMode
         // value from the environment variable if it is defined.
         internal static bool GetInvariantSwitchValue()
         {
-            bool exist;
-            bool ret = CLRConfig.GetBoolValue("System.Globalization.Invariant", out exist);
+            bool ret = CLRConfig.GetBoolValue("System.Globalization.Invariant", out bool exist);
```

Comment:
FWIW, I've generally pushed back on such refactorings.  I know that Visual Studio recommends it, and I know that it reduces the number of lines.  The problem I have is it makes it harder for me to find where things are declared.  It used to be that one only need to look at the method parameters and on the left side of lines to find where things are declared.  With such changes, it now means I may need to effectively look at all of the code in order to find declarations.  I've acquiesced when the out is part of a Try or some such method used in an if condition where the out is only used inside that if block, because then I can naturally just add that one additional place to my mental list of places to look.  I have a harder time swallowing the general case, though.